Userland/Utilities/strace.cpp

```diff
@@ -36,24 +137,328 @@ static void handle_sigint(int)
     }
 }
 
-static String format_syscall(syscall_arg_t syscall_index, syscall_arg_t arg1, syscall_arg_t arg2, syscall_arg_t arg3, syscall_arg_t res)
+static void copy_from_process(const void* source_p, void* target_p, size_t size)
```

Comment:
How about a touch of modernity?
```suggestion
static void copy_from_process(const void* source, Bytes target)
```

Reply:
Thanks! I'm still fairly new to the project and don't know all the available types yet. I'll use this as suggested.

Reply:
Done